### DISCUSSION TITLE
Container SHOULD NOT be renamed after changing stack compose and deployment

### DISCUSSION THREAD

**Discussion by baloo2** [louislam/dockge#321] [Ask for Help] [answered]
When I make changes to stack compose file and deploy the stack, the container name is changed from "container" to "randomstring_container". Is it possible to keep the original container name after deployment?

Environment: Synology DSM 7.2

**Comment by markhaines** [comment, marked as answer]
That’s not a bug - you just need to use the container_name: directive in your compose script.
